packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx

```diff
@@ -28,12 +28,18 @@ export const AssetCoinLogo = ({ symbol, assetsFiatBalances, index }: CoinLogoPro
 
     return (
         <LogoWrapper>
-            <AssetShareIndicator
-                symbol={symbol}
-                size={24}
-                percentageShare={assetPercentage}
-                index={index}
-            />
+            <Tooltip
+                content={<>{(assetPercentage ?? 0).toFixed(2)} %</>}
```

Comment:
Nitpick: Imo using template strings seems like a better option here
```suggestion
                content={`${(assetPercentage ?? 0).toFixed(2)} %`}
```